static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -63,6 +75,20 @@ const Context = ({
     ) : null;
   }
 
+  // Temporarily allow mobile platforms to make API call and "show" stacktrace link
+  if (isMobile) {
+    return (
+      <ErrorBoundary customComponent={null}>
+        <StacktraceLink
```

Comment:
If it is a mobile platform, allow the stacktrace link component to be "rendered". In actuality, we will force it to return null so just the API call is made